kitsune/products/models.py

```diff
@@ -134,6 +135,9 @@ class Topic(ModelBase):
 
     # Whether or not this topic is visible in the ui to users.
     visible = models.BooleanField(default=False)
+    # Whether or not this topic is used in the AAQ.
+    in_aaq = models.BooleanField(
+        default=False, help_text=_lazy(u'Whether this topic is shown to users in the AAQ or not.'))
```

Comment:
Out of curiosity, do you want to default these all to False? Would that break what's currently in production?